.github/workflows/ci-cd.yml

```
@@ -0,0 +1,145 @@
+    
+    steps:
+    - uses: actions/checkout@v4
+    
+    - name: Run Trivy vulnerability scanner
+      uses: aquasecurity/trivy-action@master
+      with:
+        scan-type: 'fs'
+        scan-ref: '.'
+        format: 'sarif'
+        output: 'trivy-results.sarif'
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)